src/Appwrite/Extend/Exception.php

```diff
@@ -182,6 +182,10 @@ class Exception extends \Exception
     public const GRAPHQL_NO_QUERY                  = 'graphql_no_query';
     public const GRAPHQL_TOO_MANY_QUERIES          = 'graphql_too_many_queries';
 
+    /** Timeout */
+    public const TIMEOUT_BLOCKED                    = 'timeout_blocked';
+    public const TIMEOUT                            = 'timeout';
```

Comment:
Let's be a little more explicit with these, maybe `QUERY_TIMEOUT` and `QUERY_BLOCKED`

Reply:
Done!